### PR TITLE
Initial implementation of cohort.

### PR DESCRIPTION
**Pull request by benjmarks22**

### CODE REVIEW

File: src/cohort/cohort_server.cc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include "src/cohort/cohort_server.h"
+
+#include "absl/status/statusor.h"
+#include "grpcpp/server_context.h"
+#include "src/proto/cohort.grpc.pb.h"
+#include "src/utils/grpc_utils.h"
+
+namespace cohort {
+
+using grpc::ServerContext;
```
Comment (n-triple-a) [inline src/cohort/cohort_server.cc:10]: nit: `using ::grpc::ServerContext;` to align with google style and keeping local style consistent?

File: src/cohort/cohort_server.cc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+absl::Status CohortServer::StartWork(const PrepareTransactionRequest &request) {
+  // TODO(benjmarks22): Create a fiber and start the transaction.
+  results_by_transaction_id_[request.transaction_id()]
+      .mutable_pending_response();
```
Comment (n-triple-a) [inline src/cohort/cohort_server.cc:15]: to make sure i understand correctly, database operations will be implemented in this txn right?

**Review by n-triple-a — APPROVED** [review]

File: src/cohort/cohort_server.cc (hunk 2 above)
Comment (benjmarks22) [reply src/cohort/cohort_server.cc]: Yes that's correct.

File: src/cohort/cohort_server.cc (hunk 1 above)
Comment (benjmarks22) [reply src/cohort/cohort_server.cc]: Done.

**Pull request merged**